client/electron/electron-builder.json

```diff
@@ -37,7 +37,8 @@
     "maintainer": "Jigsaw LLC",
     "target": [{
       "arch": [
-        "x64"
+        "x64",
+        "arm64"
       ],
       "target": "AppImage"
```

Comment:
`AppImage` will be deprecated in future versions, please make the change to the `deb` target instead.

Reply:
I made changes to `AppImage` and `deb`. Delete both arch from `AppImage` when it will be deprecated.